collectors/cgroups.plugin/sys_fs_cgroup.h

```diff
@@ -28,6 +28,7 @@ typedef struct netdata_ebpf_cgroup_shm_header {
     int cgroup_root_count;
     int cgroup_max;
     int systemd_enabled;
+    int pad;
```

Comment:
Is this pad not added automatically? Or are we just making it clear (in which case `__pad` might be a better name?)

Reply:
I renamed it, thanks!